core/embed/firmware/memory_R.ld

```diff
@@ -0,0 +1,81 @@
+/* TREZORv2 firmware linker script */
```

Comment:
Will this file ever be different from memory_T.ld? Should we make it a symlink until then?

Reply:
+1 for symlink

Reply:
Symlink added in [5cd212d](https://github.com/trezor/trezor-firmware/pull/2230/commits/5cd212d6b092437e82f102909ec1edd9837f82a3)